sdk/ai/azure-ai-client/azure/ai/client/operations/_patch.py

```diff
@@ -441,9 +380,13 @@ def create_agent(
                 return super().create_agent(body=body, content_type=content_type, **kwargs)
             return super().create_agent(body=body, **kwargs)
 
-        if toolset is not None:
+        tool_resources = None
```

Comment:
Absolutely not going to merge code liked this.    Pleaser review the signature and samples.